sormas-backend/src/main/java/de/symeda/sormas/backend/sample/AdditionalTestService.java

```diff
@@ -141,4 +138,10 @@ public void delete(List<String> additionalTestUuids) {
 
 		em.createQuery(cd).executeUpdate();
 	}
+
+	@Override
+	public void delete(AdditionalTest deleteme) {
+		em.remove(em.contains(deleteme) ? deleteme : em.merge(deleteme));
```

Comment:
It's super strange. I also did not manage to find out why deleteme is no longer attached.
Since we don't understand it, it could also happen for other entities, so I think we should move this code to the BaseAdoService and a comment there that the passed entity may be detached.

Reply:
ok